lms/djangoapps/bulk_email/models.py

```diff
@@ -215,6 +216,10 @@ def render_htmltext(self, htmltext, context):
         Convert HTML text body (`htmltext`) into HTML email message using the
         stored HTML template and the provided `context` dict.
         """
+        # HTML-escape string values in the context (used for keyword substitution).
+        for key, value in context.iteritems():
```

Comment:
Note that I decided to do the escaping here instead of in common/djangoapps/util/keyword_substitution.py because the utility method doesn't feel like it should know anything about where the substituted string will be used (for instance, not desirable for the plain text email).